### PR TITLE
Add OCW mirror configuration

### PR DESCRIPTION
**Pull request by markbreedlove**
This adds some minimal configuration to make sure that various directories exist on the mirror server.

### CODE REVIEW

**Comment by blarghmatey** [comment]
Looks good to me. @shaidar anything I'm missing?

File: pillar/apps/ocw.sls [hunk 1]
```diff
@@ -12,6 +12,37 @@ ocw:
...
+      - /data/prod/terms
+      - /data/prod/images
+      - /data/prod/jsp
+      - /data/prod/jw-player-free
+      - /data/prod/mathjax
+      - /data/prod/OcWeb
+      - /data/prod/scripts
+      - /data/prod/styles
+      - /data/prod/webfonts
+      - /data/prod/subscribe
```
Comment (shaidar) [inline pillar/apps/ocw.sls]: Shouldn't all those be `/data2/`?

File: salt/apps/ocw/mirror.sls [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (shaidar) [inline salt/apps/ocw/mirror.sls]: nit: Why the blank line?

Comment (markbreedlove) [reply salt/apps/ocw/mirror.sls]: no good reason! :-) I'll take it out.

File: pillar/apps/ocw.sls (hunk 1 above)
Comment (markbreedlove) [reply pillar/apps/ocw.sls]: The `/data` paths are hardcoded in https://github.com/mitocw/ocwcms/blob/25f31dd2a15b6b658b0fa59d0cd8ebb8ebe0f7c7/mirror/scripts/ocw6%20Mirror%20Scripts/create_new_snapshot.sh

`/data2` exists alongside `/data`. It's configured in the engines.conf file. I don't know why these are two separate directories, but they have been that way on the legacy systems and I'm hesitant to change things until I can say better what the separation is about.

**Pull request merged**